source/accounts/index.rst

```diff
@@ -306,49 +305,139 @@ This figure is an overview of the timeline and process for system access.
 
 .. image:: /images/access1.png
 
-Once you have a NOAA.gov email address, you can request an RDHPCS account. 
-Visit the `Account Information Management (AIM) website <https://aim.rdhpcs.noaa.gov>`_ and  request access to the RDHPCS project.  Log into AIM using your NOAA email credentials, review your profile for accuracy, and request the RDHPCS project.
+Once you have a NOAA.gov email address, you can request an RDHPCS account through the Account Iformation Managment system, AIM.
+Visit the `Account Information Management (AIM) website <https://aim.rdhpcs.noaa.gov>`_ and request access to the RDHPCS project.  Log into AIM using your NOAA email credentials, review your profile for accuracy, and request the RDHPCS project.
```

Comment:
is the trailing _ deliberate?

Reply:
Actually yes, and the syntax drives me nuts. b/c my editor adds a second backtick automatically, it blows up if I don't edit out the extra backtick and it blows up without the _ also.